### PR TITLE
Test for bitfield messages on cache miss

### PR DESCRIPTION
**Pull request by HDegroote**

### CODE REVIEW

File: test/remote-bitfield.js [hunk 1]
```diff
@@ -58,6 +58,70 @@ test('remote congituous length consistency (remote-bitfield findFirst edge case)
...
+  await t.exception(
+    async () => {
+      await empty.get(5, { timeout: 100 })
+    },
+    /REQUEST_TIMEOUT/,
+    'request on unavailable block times out (sanity check)'
+  )
+
+  // TODO: verify this is expected
+  t.is(stats.wireBitfield.rx, 1, 'No new bitfield message sent on new cache miss')
```
Comment (HDegroote) [inline test/remote-bitfield.js:121]: Is this expected? I had expected a new bitfield message, where the sparse peer communicates which blocks it now has available (both peers did send a new wireSync message to each other, so they're aware of each other's lengths)

Comment (HDegroote) [reply test/remote-bitfield.js]: confirmed it's expected, so just got rid of all those additional asserts
